docs/multicast.md

```diff
@@ -165,7 +170,7 @@ sending leave and the last multicast packet received for this group. Multicast p
 received for the leaved group after first packet of joined group is received are counted
 as overlap.
 
-The following <<Configuration>> output shows an example for the `igmp` section
```

Comment:
Please do not remove the cross references (`<<..>>`).

Reply:
have you lactually ooked at the result in github? it is simply rendered as <> so whatever that intended - it does not work.